config/locales/doc_auth/en.yml

```diff
@@ -55,9 +55,12 @@ en:
         failed: Camera failed to start, please try again.
       card_type: Try again with your driver’s license or state ID card.
       doc:
-        doc_type_check: Other forms of ID are not accepted.
-        wrong_id_type: We only accept a driver’s license or a state ID card at this
-          time. Other forms of ID are not accepted.
+        doc_type_check: Your driver’s license or state ID must be issued by a U.S. state
+          or territory. We do not accept other forms of ID, like passports or
+          military IDs.
+        wrong_id_type_html: We only accept a driver’s license or a state ID issued by a
+          U.S. state or territory. We do not accept other forms of ID, like
+          passports or military IDs. <a>Learn more about accepted IDs</a>
```

Comment:
I'm not seeing the link here?
<img width="710" alt="Screenshot 2023-09-21 at 10 54 07" src="https://github.com/18F/identity-idp/assets/6818839/7b0a39a8-8cdd-46f6-8127-df8dcb6ae02b">

Reply:
@charleyf , it's on the review issue pages.